src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -226,12 +226,12 @@ source %{
       case Op_ExtractC:
       case Op_ExtractUB:
       // Vector API specific
+        return false;
```

Comment:
Put the comment above "case Op_StoreVectorScatter" ?  Or remove "case Op_StoreVectorScatter" under "Op_VectorLoadConst" ?